.github/workflows/integration-tests.yml

```diff
@@ -29,14 +29,20 @@ jobs:
       with:
         default: 2.7.14
         versions: ${{ matrix.python-version }}
-    - name: Test with pytest
+    - name: Test with scylla
       run: |
         export EVENT_LOOP_MANAGER=${{ matrix.event_loop_manager }}
         export SCYLLA_VERSION='release:5.1'
         ./scripts/run_integration_test.sh tests/integration/standard/ tests/integration/cqlengine/
     
-    - name: Test tablets
+    - name: Test with scylla - tablets
       run: |
         export EVENT_LOOP_MANAGER=${{ matrix.event_loop_manager }}
         export SCYLLA_VERSION='release:6.0.2'
```

Comment:
Maybe change it to `6.2.0`, it is way faster then `6.0.2`

Reply:
this is unrelated to this PR